README.markdown

```diff
@@ -4523,6 +4523,8 @@ or
  ngx.req.set_uri("/foo", true)
 ```
 
+Starting from `0.10.16rc6` of this module, this function accepts an optional boolean `binary` argument to allow arbitrary binary URI data. By default, this `binary` argument is false and this function will throw out a Lua error  when the `uri` argument contains any control characters (ASCII Code 0 ~ 0x1F and 0x7F).
```

Comment:
Shall we avoid lines exceeding 80 cols here? So that it's easier to do diffs in the future. Also, you have 2 spaces before `when` BTW.